test/Microsoft.OpenApi.Tests/Models/OpenApiComponentsTests.cs

```diff
@@ -144,144 +116,80 @@ public class OpenApiComponentsTests
 
         public static OpenApiComponents BrokenComponents = new OpenApiComponents
         {
-            Schemas = new Dictionary<string, OpenApiSchema>
+            Schemas = new Dictionary<string, JsonSchema>
             {
-                ["schema1"] = new OpenApiSchema
-                {
-                    Type = "string"
-                },
+                ["schema1"] = new JsonSchemaBuilder().Type(SchemaValueType.Integer),
                 ["schema2"] = null,
                 ["schema3"] = null,
-                ["schema4"] = new OpenApiSchema
-                {
-                    Type = "string",
-                    AllOf = new List<OpenApiSchema>
-                    {
-                        null,
+                ["schema3"] = new JsonSchemaBuilder()
+                    .Type(SchemaValueType.String)
+                    .AllOf(
+                        null, // These nulls are invalid for allOf; I'm not sure what they're trying to achieve.
                         null,
-                        new OpenApiSchema
-                        {
-                            Type = "string"
-                        },
+                        new JsonSchemaBuilder().Type(SchemaValueType.String),
                         null,
                         null
-                    }
-                }
+                    )
```

Comment:
This is defining the `BrokenComponents` object for this test, but submitting `null` here is more than broken; it's completely invalid.  I'm not sure JsonSchema.Net will even support doing this, and even if it does, you'll likely get a `NullReferenceException`.